estimator/estimator_argilla.py

```diff
@@ -1,7 +1,7 @@
 import argilla as rg
 import time
-from argilla.client import api
 import pandas as pd
+from argilla.client.singleton import active_client
```

Comment:
This isn't compatible with argilla==1.19.0

If we upgrade the version, it should be updated also in the other installation method (the requirement.txt and the conda installation)

Reply:
Interesting, I tested in conda, but I just noticed that conda is using `argilla==1.20.0` while requirements.txt is using `argilla==1.19.0`. I'll get all of them to use latest version